### PR TITLE
API Prizes and Hacks blog added

### PR DESCRIPTION
**Pull request by krahulreddy**

### CODE REVIEW

File: _posts/2020-01-20-hacks.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+* Charitable organizations are obviously a good thing, especially when they actually use their funds for the intended purposes, which is typically to help people. Unfortunately, that is not always the case, and there have been situations in which charitable organizations have been labeled as corrupt and have been involved in unethical activities. Blockchain technology can help remedy this.
+* Digital IDs : More than 1 billion people worldwide face identity challenges. Creating digital IDs which would give users a way to control their digital identities.
+* Medical recordkeeping: The good news is the medical sector has already been moving away from paper for recordkeeping purposes for years.
+* However, blockchain offers even more safety and convenience. In addition to storing patient records, the patient, who possesses the key to access these digital records, would be in control of who gains access to that data.
```
Comment (shreyasseshadri) [inline _posts/2020-01-20-hacks.md]: The second point seems to be a contination. Can merge ?

File: _posts/2020-01-20-hacks.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+* Weapons tracking : One of the hot-button topics on any news network at the moment is gun control and/or weapons accountability. Blockchain could create a transparent and unchanging registry network that allows law enforcement and the federal government to track gun or weapon ownership, as well as keep a record of weapons sold privately.
+
+### AI/ML
+
+* A system which can do a check-up for emergencies and give short term advice.
+* Classifying articles or news stories as worth reading or not.
+* An app for taxi drivers- which will help them to identify which locations to concentrate and go around in loops in order to get maximum customers.
```
Comment (shreyasseshadri) [inline _posts/2020-01-20-hacks.md]: Seems to be more of an Algorithmic problem than AI

File: _posts/2020-01-20-hacks.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+* Agriculture has transformed with the advent of irrigation, but irrigation itself requires a certain level of supervision. Develop Irrigation systems which self regulate according to weather patterns.
+* Depression affects about 20% of adolescents by the time they become adults. Develop Interfaces or chatbots that can help counsel people with issues like depression or anxiety issues and suicidal tendencies.
+* Personal Health Assistant : The AI in your phone will figure out exactly when and how to wake you up given your health, sleep patterns and time constraints. It will also look into your eating habits and suggest you restaurants and dishes for keeping you in track.
+
+### Algorithms
+
+* Can you devise new algorithms to improve the control of vehicular traffic in crowded cities?
+* Bitcoin – last year it was claimed that the computing power required to keep the network running consumes as much energy as was used by 159 of the world’s nations. Come up with an implementation which solves the energy problem with your own algorithm.
```
Comment (shreyasseshadri) [inline _posts/2020-01-20-hacks.md]: Come up with an implementation which solves the energy problem with your own **Consensus** algorithm.

File: _posts/2020-01-20-hacks.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+* Digital IDs : More than 1 billion people worldwide face identity challenges. Creating digital IDs which would give users a way to control their digital identities.
+* Medical recordkeeping: The good news is the medical sector has already been moving away from paper for recordkeeping purposes for years.
+* However, blockchain offers even more safety and convenience. In addition to storing patient records, the patient, who possesses the key to access these digital records, would be in control of who gains access to that data.
+* Weapons tracking : One of the hot-button topics on any news network at the moment is gun control and/or weapons accountability. Blockchain could create a transparent and unchanging registry network that allows law enforcement and the federal government to track gun or weapon ownership, as well as keep a record of weapons sold privately.
+
+### AI/ML
+
+* A system which can do a check-up for emergencies and give short term advice.
+* Classifying articles or news stories as worth reading or not.
```
Comment (shreyasseshadri) [inline _posts/2020-01-20-hacks.md]: Not sure about _worth reading_ maybe change it to fake news or something else?

File: _posts/2020-01-20-hacks.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+* Learning Interface for All: Understanding banking instruments and financial systems remains a challenge in Tier 2-3 and rural areas due to language and smartphone usage barriers. Develop a digital solution for awareness generation for this segment to be able to understand and access relevant financial instruments.
+* Financial and Wealth Management: Wealth Management is resorted to only by the Upper-Class Indians who have significant savings to invest in Alternative Funds. Financial Planning and investment of savings by the middle class and youth is very low. Develop a solution to increase middle class and younger generation participation in Financial Markets to leverage investment of savings.
+
+### Future Mobility
+
+* Study of flow dynamics of people within a city to improve public transport infrastructure
+* Monitoring the driver to keep them alert at all times.
+* Detecting unavoidable Frontal, Rear end, T-bone, and other types of collisions to activate preventive measures.
+* mycardoeswhat.org datasets as a source of inspiration
```
Comment (shreyasseshadri) [inline _posts/2020-01-20-hacks.md]: New point needed for the dataset ?

Comment (krahulreddy) [reply _posts/2020-01-20-hacks.md]: The website has many datasets related to autonomous driving and vehicular safety. So I added in a new line.

**Pull request merged**